app/views/govuk_publishing_components/components/_single_page_notification_button.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  page ||= ''
+  data_attributes ||= {}
```

Comment:
Should the ability to pass data attributes be documented in the YML?

Reply:
have added